tests/README.md

```diff
@@ -72,11 +72,12 @@ python3 -m pytest tests/test_s3_event_config_lambda.py -v
 Run the following command from the repo root to run the integration test for the setup external storage script to check that the STS
 access has been set for a given synapse folder (in develop).
 
-This test takes in two command line arguments:
-
-- test-synapse-folder-id - synapse id of the folder to check STS access for
-- test-ssm-parameter - ssm parameter to get AWS credentials for otherwise leave blank and it will pull credentials from the environment
 
 ```shell script
-python3 -m pytest tests/test_setup_external_storage.py --test-synapse-folder-id <put_synapse_folder_id_here> --test-ssm-parameter <put_ssm_parameter_here_or_leave_blank>
+python3 -m pytest tests/test_setup_external_storage.py
+--test-bucket <put_bucket_name_here>
+--test-synapse-folder-id <put_synapse_folder_id_here>
+--test-ssm-parameter <put_ssm_parameter_here_or_leave_blank>
```

Comment:
I see that `--test-ssm-parameter` is not a new parameter for this PR, but what do you think about having this fixture return a hardcoded string literal in conftest.py, rather than passing this on the command line? We only have one SSM parameter and it doesn't change very often.(If it does, we can update the hard coded string literal.)

Reply:
Sure! That's fine with me